.github/workflows/requirements.yml

```diff
@@ -0,0 +1,31 @@
+---
+name: Update requirements file
+
+on:
+  push:
+    branches:
+      - develop
+  workflow_dispatch:
+
+jobs:
+  update-requirements:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Install Poetry
+        uses: snok/install-poetry@v1
+        with:
+          virtualenvs-create: true
+          virtualenvs-in-project: true
+      - name: Generate requirements
+        run: |
+          poetry lock
+          echo -e "# THE FILE WAS GENERATED BY POETRY, DO NOT EDIT!\n\n" > src/requirements.txt
+          poetry export --without-hashes -f requirements.txt >> src/requirements.txt
+          echo -e "# THE FILE WAS GENERATED BY POETRY, DO NOT EDIT!\n\n" > src/requirements-dev.txt
+          poetry export --dev --without-hashes -f requirements.txt >> src/requirements-dev.txt
+          echo -e "src/." >> src/requirements-dev.txt
+      - uses: stefanzweifel/git-auto-commit-action@v4
+        with:
+          commit_message: Update requirements{-dev}.txt
```

Comment:
This is optional; but you could add this to the `scripts/poetry-export.sh` bash script. The benefit of this is that if we need to run it locally, it will use a consistent commit message.

Reply:
Side note: This may technically update the poetry.lock file but you're commit message doesn't include that. That being said, it's a super minor issue.

Reply:
The message can be changed.